tar/multitape/multitape.h

```diff
@@ -172,12 +172,14 @@ int statstape_printglobal(TAPE_S *, const char *);
 int statstape_printall(TAPE_S *, const char *);
 
 /**
- * statstape_printlist(d, verbose):
+ * statstape_printlist(d, verbose, printnulls):
```

Comment:
On second thought, this would be nicer if it was `print_nulls` with an underscore.